ModTek/MergeCache.cs

```diff
@@ -175,7 +175,7 @@ public CacheEntry(string cacheAbsolutePath, string originalAbsolutePath, List<st
                 catch (Exception e)
                 {
                     Log($"\tParent JSON at path {originalAbsolutePath} has errors preventing any merges!");
-                    Log($"\t\t{e.Message}");
+                    Log(e.ToString());
```

Comment:
I would create another log method that takes a message (the one before) and an exception as a parameter, that way the message and exception are semantically combined. (e.g. one could also directly log into the HBS logger in the future or just change the formatting in a central place)

Reply:
Genius.